examples/compiler/compiler.rs

```diff
@@ -1,6 +1,74 @@
-pub trait CompilerDatabase: salsa::Database {
-    fn interner(&self) -> &Interner;
+use std::sync::Arc;
+
+use crate::{interner::Interner, values::*};
+
+#[salsa::query_group(CompilerDatabase)]
+#[salsa::requires(Interner)]
```

Comment:
There's no need for `requires` clause, because there's `: Interner` "public" inheritance anyway.

Reply:
Fixed.